src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
-body {
-  margin: 0;
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white-accent: #fff;
+
+.product-card {
+  font-family: Roboto, Arial, Helvetica, 'Segoe UI', sans-serif, system-ui;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to avoid specifying generic font family names unless they are actually needed. In this case, 'Arial', 'Helvetica', 'Segoe UI', and 'system-ui' might not be necessary if 'Roboto' and 'sans-serif' cover the desired font styles.